bcheng-ms/azure-search-openai-demo
# adding okta auth

## Purpose
Adding Okta authentication

## Does this introduce a breaking change?
It should not.  Okta is on a different path
```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Try to login


* Test the code
Hit the /protected url and see if you are forwarded to login
```
```

## What to Check
Verify that the following are valid
Your name should show up on the protected page after its logged in.
